backend/src/utils/types.ts

```diff
@@ -0,0 +1,40 @@
+type User = {
+    id : string,
+    role : string,
+    email : string
+}
+
+type Dish = {
+    id : string,
+    qid : number,
+    registered : Date,
+    type: string
+}
+
+// need more info on returned type
+type Transaction = {
+    id : string,
+    dishID : string,
+    userID: string,
+    returned : any,
+    timestamp : Date
+}
+
+enum DishStatus {
+    overdue =  "Overdue",
+    inUse = "In Use",
+    returned = "Returned",
+    broken = "Broken",
+    lost = "Lost"
```

Comment:
can you update the enums to all use camelCase or snake_case so that we keep the consistency?